mininet/fault_controllers/README.md

```diff
@@ -8,16 +8,57 @@ At this point in time three different fault controller are implemented:
 - RandomLinkFaultController, which inserts pre-defined faults into randomly selected links
 - MostUsedFaultController, which inserts pre-defined faults into the busiest links
 
-For more details on how to use these fault controller see the [FaultControllersREADME](FaulControllersREADME.md) file.
-For more implementation details see the [Documentation](Documentation.md) file.
+For more details on how to use these fault controller see the [FaultControllersREADME](FaultyControllers.md) file.
+For more implementation details see the [Documentation](Faultynet-Documentation.md) file.
 
 ## Installation
 Faultynet requires `libcgroup1` to be installed on your system. This package is (as of time of writing) not present in apt for ubuntu 24.04, but you can still find .deb files on the internet to install.
 
-Faultynet also needs legacy cgroups. You need to add `SYSTEMD_CGROUP_ENABLE_LEGACY_FORCE=1 systemd.unified_cgroup_hierarchy=0` to your kernel parameters and reboot. If you are using grub, you can add this parameters by appending them to `GRUB_CMDLINE_LINUX_DEFAULT` in `/etc/default/grub` and running `sudo grub-mkconfig -o /boot/grub/grub.cfg`
+Faultynet also needs legacy cgroups. You need to add `SYSTEMD_CGROUP_ENABLE_LEGACY_FORCE=1` and `systemd.unified_cgroup_hierarchy=0` to your kernel parameters and reboot. 
+
+If you are using GRUB, you can add this parameters by appending them to `GRUB_CMDLINE_LINUX_DEFAULT` and `GRUB_CMDLINE_LINUX` in `/etc/default/grub` and running `sudo grub-mkconfig -o /boot/grub/grub.cfg`
+```bash
+sudo vim /etc/default/grub
+```
+Add the systemd.unified_cgroup_hierarchy=0 parameter inside the quotes
+```bash
+GRUB_CMDLINE_LINUX_DEFAULT="systemd.unified_cgroup_hierarchy=0"
+GRUB_CMDLINE_LINUX="systemd.unified_cgroup_hierarchy=0"
+```
+Update GRUB to apply the changes:
+```bash
+sudo grub-mkconfig -o /boot/grub/grub.cfg
+```
+Reboot your system to apply. 
```

Comment:
Two things:
- I don't think you should add this to `GRUB_CMDLINE_LINUX` as well, as this applies it to the recovery boot as well. If something fails, we want the recovery boot to at least work. Just adding it to `GRUB_CMDLINE_LINUX_DEFAULT` suffices
- You **must** add `SYSTEMD_CGROUP_ENABLE_LEGACY_FORCE=1` as well, as newer versions of systemd will just refuse to boot with only `systemd.unified_cgroup_hierarchy=0` set. I can say from experience that it is very annoying to be locked out of ones system because you only added one kernel parameter. It's even more annoying when you only have remote access to the machine.